bot.js

```diff
@@ -1169,22 +1208,26 @@ function command(message) {
                     );
 
                     // Grab the report of how much time is left from the response, and parse it into a string
-					try {
```

Comment:
Woah, woah, woah - How the heck did *tabs* get in here!?

Goddammit... Um, me? VSCode?

Yeah, let's go with VSCode.